### PR TITLE
Capture realtime metrics in health report

### PR DESCRIPTION
**Pull request by anjalshireesh**
## Description

Capture realtime metrics in health report

## Motivation and Context

Will be useful for debugging

## How to test this PR?

- Generate health report for a cluster using `mc support diag --airgap` (must use mc built with https://github.com/minio/mc/pull/4618)
- Extract it and verify that it contains a `metrics` node under `minio -> info` that contains realtime metrics
- Verify that the hostnames / ip-addresses under this node are anonymized

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Optimization (provides speedup with no functional changes)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [ ] Fixes a regression (If yes, please add `commit-id` or `PR #` here)
- [ ] Unit tests added/updated
- [ ] Internal documentation updated
- [ ] Create a documentation update request [here](https://github.com/minio/docs/issues/new?label=doc-change,title=Doc+Updated+Needed+For+PR+github.com%2fminio%2fminio%2fpull%2fNNNNN)

### CODE REVIEW

File: cmd/admin-handlers.go [hunk 1]
```diff
@@ -2103,6 +2103,27 @@ func fetchHealthInfo(healthCtx context.Context, objectAPI ObjectLayer, query *ur
 		}
 	}
 
+	// collect all realtime metrics execpt disk
```
Comment (shtripat) [inline cmd/admin-handlers.go]: %s/execpt/except/g

Comment (anjalshireesh) [reply cmd/admin-handlers.go]: thanks, fixed

**Review by klauspost — APPROVED** [review]
LGTM

**Review by shtripat — APPROVED** [review]
LGTM. Verified

**Pull request merged**